odoo/addons/base/res/res_config.py

```diff
@@ -366,14 +366,18 @@ class MyConfigWizard(models.TransientModel):
             ``execute`` triggers the immediate installation of the module named 'XXX' 
             if the field has the integer value ``1``.
 
+        *   For a field with no specific prefix BUT an attribute 'config_parameter',
+            ``execute``` will save its value in an ir.config.parameter (global setting for the
+            database).
```

Comment:
It seems a bit weird to require a prefix for the other classes of fields but not this one.
I can understand that you don't want to rename all fields, but that makes the API of the mixin suck a bit...
(Maybe off topic) What about releasing the prefix convention on the other types?

Reply:
Actually, I don't care, but:
1) adding a prefix `config_` won't add anything in terms of expressiveness since we already MUST use the `config_param=` parameter (and this parameter is sufficient to identify fields that have to be stored as an `ir.config_parameter`)
2) furthermore, it will trigger a large refactoring of all the `res.config` + all the associated views for the sake of our perfectionnist minds... 
3) we could encourage this convention for future things with a warning or something ?